common/changes/office-ui-fabric-react/anihan-hover-card-pref-fix_2017-08-28-23-48.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "HoverCard perf fix",
```

Comment:
nit, this goes into public release notes. Would you like to read "HoverCard perf fix" in there? 

A better comment that people would appreciate more would explain what scenario you're impacting. It doesn't have to be super specific; just not totally general like "fixed thing". For example:

"HoverCard: improved performance of rendering initial load animation."